packages/opentelemetry-node/src/spanprocessor.ts

```diff
@@ -23,7 +23,7 @@ export const SENTRY_SPAN_PROCESSOR_MAP: Map<SentrySpan['spanId'], SentrySpan> =
 export class SentrySpanProcessor implements OtelSpanProcessor {
   public constructor() {
     addGlobalEventProcessor(event => {
-      const otelSpan = trace.getActiveSpan() as OtelSpan;
+      const otelSpan = trace && trace.getActiveSpan && (trace.getActiveSpan() as OtelSpan | undefined);
```

Comment:
Snuck this change in as it seems that `getActiveSpan` is not defined for some versions - will investigate more but at least this unhandled errors in production.